aeron-tools/src/main/java/uk/co/real_logic/aeron/tools/TLRandom.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 Kaazing Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.co.real_logic.aeron.tools;
+
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class TLRandom
```

Comment:
If using abbreviations do it with only the first letter capitalised. Or better just spell out for clarity.